src/poetry/installation/installer.py

```diff
@@ -351,7 +351,7 @@ def _write_lock_file(self, repo: LockfileRepository, force: bool = False) -> Non
 
             if updated_lock:
                 self._io.write_line("")
-                self._io.write_line("<info>Writing lock file</>")
+                self._io.write_line("<info>Lock file written</>")
```

Comment:
Well then, I guess that a new IO section would be needed and the https://github.com/python-poetry/poetry/blob/fe9f85baa1a533eb66cde1ee8f76fcf5386dbae2/src/poetry/packages/locker.py#L251-L259
snippet would probably have to be rewritten to
```py
     if self._should_write(lock): 
	     section = ...
         section.write_line("Writing lock file")
         self._write_lock_data(lock) 
         section.write_line("Lock file written")
         return True 
```

Is it worth the cost? It would require introducing a side effect in `_write_lock_file` or adding some extra logic to `set_lock_data`. IO writes are quite fast in contrary to installation operations; the only advantage it would bring is that it could inform about a potentially (but very rarely) erroneous action taken (in case the write fails) useful as a context for a traceback following.

Reply:
Ok, then maybe we could do something like `Writing lock file... Done.` where `Done.` appears after the operation is finished?

Reply:
> Ok, then maybe we could do something like `Writing lock file... Done.` where `Done.` appears after the operation is finished?

I think that would be inconsistent with the rest of the UI. Also, would it address the issue with additional logic/new `_write_lock_file` side effects somehow?